docs/docs-content/getting-started/getting-started.md

```diff
@@ -8,17 +8,30 @@ sidebar_custom_props:
 tags: ["getting-started"]
 ---
 
-This page gives an overview of getting started with Spectro Cloud Palette quickly. We introduce Palette's complimentary
-subscription plans, features, workflow, and user experience session to our users.
+This page gives an overview of how to get started with Spectro Cloud Palette and begin leveraging its Kubernetes
+full-stack management at scale. Palette's unique capabilities provide end-to-end declarative cluster management, cluster
+monitoring and reconciliation, as well as adherence to the highest security standards. It provides support for public
+cloud providers, data centers, bare metal and edge, so you can use Palette no matter the complexity of your production
+environments.
 
-The first step towards adopting Palette to your organization is to
+The first step towards adopting Palette in your organization is to
 [create a login](https://www.spectrocloud.com/get-started). The
 [Palette Free Tier](https://www.spectrocloud.com/free-tier) gives our users the opportunity to get a first impression of
 our product before making a purchase decision.
```

Comment:
```suggestion
[create a login](https://www.spectrocloud.com/get-started). The
[Palette Free Tier](https://www.spectrocloud.com/free-tier) allows you to experience the benefits of Palette first-hand.
```
Added active voice and removed the "purchase" topic to instead focus on the user experience 😄